src/ResourceManager/IotHub/Commands.IotHub/help/Add-AzureRmIotHubEventHubConsumerGroup.md

```diff
@@ -39,7 +38,7 @@ Adds a new consumergroup named "myconsumergroup" to the eventhub for operations
 ## PARAMETERS
 
 ### -EventHubConsumerGroupName
-Name of the EventHub ConsumerGroup that you want to add.
+EventHubConsumerGroupName.
```

Comment:
Please use complete sentences in the help descriptions.  This applies acrosss the board to all help files